ui/perfherder/compare/CompareSortButton.jsx

```diff
@@ -24,11 +24,14 @@ export default class CompareSortButton extends React.Component {
   };
 
   render() {
-    const { column, onChangeSort } = this.props;
+    const { column, onChangeSort, measurementUnit } = this.props;
```

Comment:
The measurement unit isn't something we can apply on all `CompareSortButton`s. It's specific to numeral test values.

I think we need to refactor the table configs from `CompareTable` component, so that we can directly provide the button name at construction time.